CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing guidelines
+
+## Code formatting
+
+- For go code, simply use go conventions (can be formatted via `go fmt`)
+- Avoid _when possible_ lines of 80+ chars
+- Every file should end with an empty line
+
+## Git
+
+- Branch names must follow this pattern: `<type>/name-of-related-issue`
+- Commit messages must be 50 chars max (add a description if necessary)
+- Commit messages must start with a verb in the present simple, such as `Add ...`, `Implement ...`, `Fix ...`
+
+## Pull requests & Merge policy
```

Comment:
[Suggestion] We could add here some rules we established in the Discord:

- PR Status: draft ➡️ WIP (no reviews); open ➡️ ready for review
- We can request specific reviewers, but anyone can review any PR
- No push on someone else's PR
- PR mergeability: all comments must have been addressed before merge (change in code / agreement in thread / issue create if out of scope)